### ISSUE TITLE
Exception in 'update_video_info': 'dict' object has no attribute 'to_yaml'

### ISSUE THREAD

**Issue by marillat** [cdgriffith/FastFlix#598]
I'm sorry to bother you, but I see this error (see the end of this issue)  when I open a video file and I'm unable to find what is wrong so I need your help.
Here is debian packages installed vs packages installed with venv. 
|Package|venv version|Debian version
|---|---|---|
|appdirs|1.4.4|1.4.4
|chardet|5.1.0|5.2.0
|charset-normalizer|3.3.2|3.3.2
|colorama|0.4.6|0.4.6
|coloredlogs|15.0.1|15.0.1
|iso639-lang|0.0.9|0.0.9
|mistune|2.0.5|3.0.2
|packaging|24.1|24.1
|pathvalidate|2.5.2|3.2.0
|psutil|5.9.8|5.9.8
|pydantic|2.8.2|2.4.2
|pydantic_core|2.20.1|2.11.0
|PySide6|6.7.2|6.6.2
|python-box|6.1.0|6.1.0
|requests|2.32.3|2.32.3
|reusables|0.9.6|0.9.6
|-|-|-|
|ruamel.yaml |0.18.6|0.18.6+ds-3
|ruamel.yaml.clib|0.2.8|0.2.8+ds-2

```
2024-08-11 16:02:58 christian.marillat.net fastflix[18208] ERROR Exception in 'update_video_info': 'dict' object has no attribute 'to_yaml' - args: (<fastflix.widgets.main.Main(0x107f1640, name="Main") at 0x7fe251180040>,)
2024-08-11 16:02:58 christian.marillat.net fastflix[18208] ERROR Could not load video /home/marillat/file_example_MP4_1920_18MG.mp4
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/fastflix/widgets/main.py", line 1109, in open_file
    self.update_video_info()
  File "/usr/lib/python3/dist-packages/reusables/wrappers.py", line 293, in wrapper
    raise err
  File "/usr/lib/python3/dist-packages/reusables/wrappers.py", line 285, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/fastflix/widgets/main.py", line 1563, in update_video_info
    self.video_options.new_source()
  File "/usr/lib/python3/dist-packages/fastflix/widgets/video_options.py", line 184, in new_source
    self.audio.new_source(self.audio_formats)
  File "/usr/lib/python3/dist-packages/fastflix/widgets/panels/audio_panel.py", line 350, in new_source
    new_item = Audio(
               ^^^^^^
  File "/usr/lib/python3/dist-packages/fastflix/widgets/panels/audio_panel.py", line 83, in __init__
    self.widgets.audio_info.setToolTip(audio_track.raw_info.to_yaml())
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'to_yaml'
2024-08-11 16:03:00 christian.marillat.net fastflix-core[18199] DEBUG Conversion worker shutting down
```

**Comment by cdgriffith** [comment]
Should be fixed in 5.8.1! https://github.com/cdgriffith/FastFlix/releases/tag/5.8.1

**Comment by marillat** [comment]
Thanks. Work fine.
